deploy/seccompagent.yaml

```diff
@@ -65,6 +68,8 @@ spec:
           mountPath: /host/seccomp
         - name: run
           mountPath: /run
+        - name: cgroup
+          mountPath: /sys/fs/cgroup
```

Comment:
Just curious, I see the code uses `/proc/pid/cgroup` and not this path at all. Why is this needed? Is this needed for that proc file to show the correct content? Or for the libct cgroup freeze to work?

Reply:
Yes, the libct cgroup freeze will freeze the container by using /sys/fs/cgroup/freezer (or equivalent, on cgroup-v2 or hybrid cgroups).

I bind mount /sys/fs/cgroup from the host instead of taking the one from the container because:
- this makes sure it is mounted in RW mode, so libcontainer can write "frozen" to it
- in case the container runtime uses cgroupns, so we have the full cgroupfs view by picking the mount from the host (but in the end, we have to make sure that cgroupns is disabled, otherwise /proc/$pid/cgroup would not have absolute paths...)

Reply:
Cool. Can you please add a comment saying this path is needed for the freeze handler only?